### ISSUE TITLE
[Question] Chunks from transcription always empty text

### ISSUE THREAD

**Issue by quantuminformation** [xenova/transformers.js#268]
This example works fine: 
![image](https://github.com/xenova/transformers.js/assets/216566/970c3828-8fbf-4539-843d-a96554c72f4b)

But ATM I am sending Float32 to the worker here (i also confirm the audio is valid by playing it back)
https://github.com/quantuminformation/coherency/blob/main/components/audio-recorder.js#L104

But after transcribing here:
https://github.com/quantuminformation/coherency/blob/main/worker.js#L140

my chunks only contain `""`

![image](https://github.com/xenova/transformers.js/assets/216566/04588e73-2ee5-4f39-a145-f4e87c392ba1)

![image](https://github.com/xenova/transformers.js/assets/216566/febe2809-0fa7-4e21-8b71-d5724a391644)

any ideas where my setup is going wrong?

**Comment by xenova** [comment]
Are you setting the sampling rate to 16000Hz? That might be something which is missing.

**Comment by xenova** [comment]
I think you can fix it by setting sampleRate here:

https://github.com/quantuminformation/coherency/blob/fb259ce35547aef8e4ad894fdcb2d6b6d637ed6d/components/audio-recorder.js#L20

e.g., 

```js
const audioContext = new (window.AudioContext || window.webkitAudioContext)({
    sampleRate: Constants.SAMPLING_RATE,
});
```

**Comment by quantuminformation** [comment]
> Are you setting the sampling rate to 16000Hz? That might be something which is missing. If I remember correctly, it defaults to 44.1kHz or 48kHz, so you need to override this.
> 
> e.g., in whisper-web:
> 
> https://github.com/xenova/whisper-web/blob/653f9790bd4cb9e23f6e094c4ba9454b190fe3fe/src/components/AudioManager.tsx#L182-L184

yeah, using 16000 : https://github.com/quantuminformation/coherency/blob/main/constants.js#L31

btw what do you think about the skipping react and only using vanilla

**Comment by xenova** [comment]
> yeah, using 16000 : https://github.com/quantuminformation/coherency/blob/main/constants.js#L31

But are you using it? The only place it is used is in routes/process-map.js
![image](https://github.com/xenova/transformers.js/assets/26504141/2ade99fd-9d35-454e-ad20-b502c22ceaef)

**Comment by quantuminformation** [comment]
yeah thx, fixed that, same issue, must be smth else im missing

**Comment by quantuminformation** [comment]
https://github.com/quantuminformation/coherency/commit/f046c0403d65469e168a2d647a32c2aa039d5c65

**Comment by quantuminformation** [comment]
I also added the missing blob fix, but still get no chunks back except ""

I wonder if there is an issue with cdn version, maybe I should try npm?

**Comment by xenova** [comment]
> I wonder if there is an issue with cdn version, maybe I should try npm?

the CDN and npm versions should give the same result. If you're still stuck, perhaps you can work backwards from [whisper web](https://github.com/xenova/whisper-web), since the audio processing is done correctly there.

**Comment by quantuminformation** [comment]
ill probably do that thanks, i think the project is on hold now due to my delays
